src/components/gauge/gauge.tsx

```diff
@@ -148,87 +160,84 @@ export class Gauge implements GxComponent {
             height: `${this.minimumSize}px`,
             width: `${this.minimumSize}px`
           }}
-        />
-        {this.showValue ? (
-          <span
-            class="marker"
+        >
+          <svg width="100%" height="100%" viewBox="0 0 100 100">
+            <circle
+              r={radius}
+              cx="50%"
+              cy="50%"
+              stroke={"rgba(0, 0, 0, 0.2)"}
+              fill="none"
+              stroke-width={`${this.thickness / 2}%`}
+            />
+            {svgRanges}
+          </svg>
+          <div
+            class="gaugeContainer"
             style={{
-              display: this.showValue ? "" : "none",
               height: `${this.minimumSize}px`,
-              width: `${this.thickness / 8}px`,
-              transform:
-                this.calcPercentage() >= 100
-                  ? "rotate(359.9deg)"
-                  : this.calcPercentage() > 0
-                  ? `rotate(${ONE_PERCENT_OF_CIRCLE_DREGREE *
-                      this.calcPercentage()}deg)`
-                  : "rotate(0.5deg)"
+              width: `${this.minimumSize}px`
             }}
-          >
-            <div
-              class="indicator"
+          />
+          {this.showValue ? (
+            <span
+              class="marker"
               style={{
-                height: `${(this.minimumSize * this.thickness) / 100}px`
+                display: this.showValue ? "" : "none",
+                height: `${this.minimumSize}px`,
+                width: `${this.thickness / 8}px`,
+                transform:
+                  this.calcPercentage() >= 100
+                    ? "rotate(359.9deg)"
+                    : this.calcPercentage() > 0
+                    ? `rotate(${ONE_PERCENT_OF_CIRCLE_DREGREE *
+                        this.calcPercentage()}deg)`
+                    : "rotate(0.5deg)"
               }}
-            />
-          </span>
-        ) : (
-          ""
-        )}
-        <div class="gauge">
-          {this.showValue ? (
-            <div>
-              <span class="current-value">{`${this.value}`}</span>
-              <span>{`${this.minValue}`}</span>
-              <span>{`-`}</span>
-              <span>{`${
-                this.maxValue == undefined ? this.maxValueAux : this.maxValue
-              }`}</span>
-            </div>
+            >
+              <div
+                class="indicator"
+                style={{
+                  height: `${(this.minimumSize * this.thickness) / 100}px`
+                }}
+              />
+            </span>
           ) : (
             ""
           )}
+          <div class="gauge">
+            {this.showValue ? (
+              <div>
+                <span class="current-value">{`${this.value}`}</span>
+                <span>{`${this.minValue}`}</span>
+                <span>{`-`}</span>
+                <span>{`${
+                  this.maxValue == undefined ? this.maxValueAux : this.maxValue
```

Comment:
```suggestion
                  this.maxValue === undefined ? this.maxValueAux : this.maxValue
```

Reply:
MInor change applied in 65fb97b! Check it out!
Tell me what do u think 😄